### PR TITLE
documentation(EJ2-56560): Documented see also for how to get the RTE …

### PR DESCRIPTION
**Pull request by VinithaJeyakumar**
documentation(EJ2-56560): Documented see also for how to get the RTE value on the controller side.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
+node('content')
```
Comment (PrinceOliver) [inline Jenkinsfile:1]: Why is this changes included

Comment (VinithaJeyakumar) [reply Jenkinsfile:1]: CI has not started for this PR. So reported the issue to build team and they asked me to commit these jenkins file. hence added this

**Review by PrinceOliver — APPROVED** [review]
